core/include/usGetBundleContext.h

```diff
@@ -29,10 +29,12 @@
 #endif
 
 #include <usBundleContext.h>
-#include <usBundleUtils.h>
 
-#include <memory>
-#include <cstring>
+#define GETBUNDLECONTEXT_FCNPREFIX _us_get_bundle_context_instance
+#define GETBUNDLECONTEXTINSTANCEFCN(Prefix, BundleName) GETBUNDLECONTEXTINSTANCEFCN_(Prefix, BundleName)
+#define GETBUNDLECONTEXTINSTANCEFCN_(Prefix, BundleName) Prefix##_##BundleName
+
+extern "C" us::BundleContextPrivate* GETBUNDLECONTEXTINSTANCEFCN(GETBUNDLECONTEXT_FCNPREFIX, US_BUNDLE_NAME)();
```

Comment:
It is a good idea to clean that up and define some common macros. We can do some more cleanup and simplify the `GetBundleContext` method, see 89dbe3f0d10af9d80e8e8b2388a67e1274700292 for my suggestions. If you agree with these changes, you can cherry-pick that commit and apply it to the HEAD of your branch.